### PR TITLE
Add About View & related tests

### PR DESCRIPTION
**Pull request by Nitish145**
### This PR contains the following changes:
- Add API utilities.
- Add About ViewModel bindings.
- Add About View & related tests.

### Screenshots / Video Recording
About View

### CODE REVIEW

**Comment by Nitish145** [comment]
@nitin10s can you please first review this PR so that other API related PRs could be sent..

**Comment by Nitish145** [comment]
@nitin10s This is actually a web view, so this will actually be needed to rebuild as flutter view.

**Comment by Nitish145** [comment]
@nitin10s

**Comment by nitin10s** [comment]
ok, everything else LGTM 👍

**Comment by Nitish145** [comment]
@tachyons anything else over here ?

File: lib/utils/api_utils.dart [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+      case 500:
+        print(response.body);
+        throw InternalServerErrorException(response.body);
+      case 503:
+        print(response.body);
+        throw ServiceUnavailableException(response.body);
+      default:
+        print(response.body);
+        throw FetchDataException(
+          'Error occured while Communication with Server with StatusCode : ${response.statusCode}',
```
Comment (tachyons) [inline lib/utils/api_utils.dart:139]: @Nitish145 typo

File: lib/utils/app_exceptions.dart [hunk 2]
```diff
@@ -0,0 +1,57 @@
+class AppException implements Exception {
+  final _message;
+  final _prefix;
+
+  AppException([this._message, this._prefix]);
+
+  String get message => _message;
+
+  String get pefix => _prefix;
```
Comment (tachyons) [inline lib/utils/app_exceptions.dart:9]: intentional?

Comment (Nitish145) [reply lib/utils/app_exceptions.dart:9]: We may require the error message, so yes intentional..

File: lib/utils/api_utils.dart (hunk 1 above)
Comment (Nitish145) [reply lib/utils/api_utils.dart:139]: Oops! Will update..

Comment (Nitish145) [reply lib/utils/api_utils.dart:139]: `Error occured while communicating with server with status code`.. is this ok ?

File: lib/utils/app_exceptions.dart (hunk 2 above)
Comment (tachyons) [reply lib/utils/app_exceptions.dart:9]: I mean using the var pefix instead of prefix

Comment (Nitish145) [reply lib/utils/app_exceptions.dart]: Ah Shit! Here we go again.. this is a typo.. :|

Comment (Nitish145) [reply lib/utils/app_exceptions.dart]: Corrected @tachyons

**Pull request merged**